### ISSUE TITLE
Feature to jump to the first and last tabs (opened files)?

### ISSUE THREAD

**Issue by JAT86** [notepad-plus-plus/notepad-plus-plus#4063]
Is there a way to jump to the opened files at the first and last tabs? I have several files I need to work on that are simultaneously open, and there are no arrow icons to jump to the last or first tabs. Navigating is difficult since I am forced to use the mouse wheel or the next and previous arrow buttons just to get to these files.

![npptabfeaturerequest](https://user-images.githubusercontent.com/30237173/34750909-944ea6bc-f5e4-11e7-943c-0901ab33ebb2.png)

**Comment by sasumner** [comment]
@JAT86  Kind of obvious but how about clicking on the tab you want to go to with the mouse?  Or perhaps the LocationNavigate plugin helps you?  Or, if you have 9 or less tabs, the View (menu) -> Tab -> ____ could be of assistance (assign Ctrl+1 to "tab1", Ctrl+2 to "tab2", etc.

Your arrows in your screenshot don't make much sense for the functionality you describe, as I would think they would walk a user through all of the tabs in order, not jump between first and last.  Maybe something like this would suit your desired functionality better (where I've marked what you want with blue splashes; sorry for the crude imagery):

![image](https://user-images.githubusercontent.com/30118311/34773757-d596b1e6-f5da-11e7-9375-bf939203c213.png)

**Comment by JAT86** [comment]
Yes, I was referring to those end arrows you marked in blue. Thanks for adding the photo.

**Comment by SinghRajenM** [comment]
@JAT86  you can use ```CTRL + SHIFT + MOUSEWHEEL``` on tabs. It will switch to first and tab.

![screencapture](https://user-images.githubusercontent.com/14791461/35479734-251e8570-0424-11e8-9d79-fff1eddc6898.gif)

**Comment by sasumner** [comment]
@SinghRajenM Wow, that is fairly obscure...but it works!  Nice one.

**Comment by alankilborn** [comment]
Suggest CLOSING this issue as @SinghRajenM pointed out the solution.
